client/web.go

```diff
@@ -177,15 +177,22 @@ func (srv *HttpService) CreateSubscription(c *gin.Context) {
 		return
 	}
 
-	if err := validateRequest(&req, endpoint.Type); err != nil {
+	if err = validateRequest(&req, endpoint.Type); err != nil {
 		logger.Error(err)
 		c.JSON(http.StatusBadRequest, nil)
 		return
 	}
 
+	spec, err := json.Marshal(req.Params)
+	if err != nil {
+		logger.Error(err)
+		c.JSON(http.StatusInternalServerError, nil)
+		return
+	}
+
 	js := &store.JobSpec{
 		Job:  req.JobID,
-		Spec: req.Params.FluxMonitor,
+		Spec: spec,
```

Comment:
Nice. So we are storing the whole actual JobSpec here, right? Both for RL and FM.

Reply:
Correct